Minor doc updates. (#14409)
apache/druid: docs/configuration/index.md

|`druid.coordinator.kill.durationToRetain`|Only applies if you set `druid.coordinator.kill.on` to `true`. This value is ignored if `druid.coordinator.kill.ignoreDurationToRetain` is `true`. Valid configurations must be a ISO8601 period. Druid will not kill unused segments whose interval end date is beyond `now - durationToRetain`. `durationToRetain` can be a negative ISO8601 period, which would result in `now - durationToRetain` to be in the future.<br /><br />Note that the `durationToRetain` parameter applies to the segment interval, not the time that the segment was last marked unused. For example, if `durationToRetain` is set to `P90D`, then a segment for a time chunk 90 days in the past is eligible for permanent deletion immediately after being marked unused.|`P90D`|
|`druid.coordinator.kill.ignoreDurationToRetain`|A way to override `druid.coordinator.kill.durationToRetain` and tell the coordinator that you do not care about the end date of unused segment intervals when it comes to killing them. If true, the coordinator considers all unused segments as eligible to be killed.|false|

> |`druid.coordinator.kill.maxSegments`|Kill at most n unused segments per kill task submission. This only applies if you set `druid.coordinator.kill.on` to `true` and must be greater than 0 then.|100|

Reviewer: ```suggestion
|`druid.coordinator.kill.maxSegments`|The number of unused segments to kill per kill task submission. This number must be greater than 0. This only applies when `druid.coordinator.kill.on=true`.|100|
```
Suggested rewording to make the property value more obvious and remove `n`. Though I wonder if "per kill task submission" should really mean "per kill task".

Author: yep, per kill task sounds much better. Adjusted it, thanks.

> |`druid.coordinator.kill.maxSegments`|The number of unused segments to kill per kill task submission. This number must be greater than 0. This only applies when `druid.coordinator.kill.on=true`.|100|

Author: ```suggestion
|`druid.coordinator.kill.maxSegments`|The number of unused segments to kill per kill task. This number must be greater than 0. This only applies when `druid.coordinator.kill.on=true`.|100|
```
